mmaction/models/recognizers/recognizer2d.py

```diff
@@ -70,20 +72,31 @@ def _do_test(self, imgs):
             x = x.squeeze(2)
             num_segs = 1
 
+        if self.feature_extraction:
+            # perform spatial pooling
+            avg_pool = nn.AdaptiveAvgPool2d(1)
+            x = avg_pool(x)
+            # squeeze dimensions
+            x = x.reshape((batches, num_segs, -1))
+            # temporal average pooling
+            x = x.mean(axis=1)
+            return x
```

Comment:
To make users more flexible to handle these features, maybe we can return x directly and save them without avg_pool reshape mean. Is there some flexible method?